StacksOfPlates.py

```diff
@@ -0,0 +1,48 @@
+class Stack:
+    def __init__(self, cap):
+        self.stack = []
+        self.cap = cap
```

Comment:
Avoid shortening variable names like this, especially if it's not a common abbreviation. A longer, more descriptive variable name is almost always preferred to a shorter variable name that may cause confusion.